programmers/난이도별/level03.등굣길/sangmandu.py

```diff
@@ -0,0 +1,62 @@
+'''
+https://programmers.co.kr/learn/courses/30/lessons/42898
+등굣길
+stack 사용
+'''
+
+def solution(m, n, puddles):
+    b = [[0] * (m + 1) for _ in range(n + 1)]
+
+    for x, y in puddles:
+        if x <= m and y <= n:
+            b[y][x] = -1
+
+    b[1][1] = 1
+    for i in range(1, n + 1):
+        for j in range(1, m + 1):
+            if b[i][j] == -1:
+                b[i][j] = 0
+                continue
+            if i != 1:
+                b[i][j] += b[i - 1][j]
+            if j != 1:
+                b[i][j] += b[i][j - 1]
```

Comment:
```suggestion
            b[i][j] += (b[i - 1][j] + b[i][j - 1])
```

`b` 배열이 0으로 초기화 되어있기 때문에 if문 처리를 하지 않아도 통과됩니다.

Reply:
감사합니다